tools/install.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 #
-# This will:
-#   1. Set up the LORIS DB schema
-#   2. Log the installation in the logs directory
-# This will only install the database components and LORIS config file.
+# This script will:
+#   1. Install the directory structure with appropriate permissions
+#   2. Configure apache (optional)
+#   3. Log these steps in the logs/ directory (if writable)
```

Comment:
```suggestion
#   3. Log these steps in the logs/ directory
```
This script creates the `logs/` directory so I don't see why it wouldn't be writable.